packages/nextra-theme-docs/tsup.config.ts

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
   entry: ['src/index.tsx'],
   format: 'esm',
   dts: true,
+  external: ['nextra'],
```

Comment:
@promer94 this fixes #1031 :D

Reply:
I think we should nextra as theme's `peerDependencies`